src/blocks/progress-bar/attributes.js

```diff
@@ -0,0 +1,35 @@
+const attributes = {
+	text: {
+		type: 'string'
```

Comment:
These attributes can have value source as HTML instead of being saved in the HTML comments.